homeassistant/components/denonavr/config_flow.py

```diff
@@ -139,17 +144,20 @@ async def async_step_select(self, user_input=None):
             step_id="select", data_schema=select_scheme, errors=errors
         )
 
-    async def async_step_confirm(self, user_input=None):
+    async def async_step_confirm(
+        self, user_input: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, Any]:
         """Allow the user to confirm adding the device."""
         if user_input is not None:
             return await self.async_step_connect()
 
         return self.async_show_form(step_id="confirm")
 
-    async def async_step_connect(self, user_input=None):
+    async def async_step_connect(
+        self, user_input: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, Any]:
         """Connect to the receiver."""
         connect_denonavr = ConnectDenonAVR(
-            self.hass,
```

Comment:
This needs an `get_async_client(hass)` at the end

Reply:
Same here, done